packages/@okta/i18n/src/properties/login.properties

```diff
@@ -710,6 +710,10 @@ password.expiring.subtitle.specific = When password expires you will be locked o
 password.expired.custom.submit = Go to {0}
 password.expired.custom.subtitle = This password is set on another website. Click the button below to go there and set a new password.
 
+## Custom Password expiring soon.
+password.expiring.soon.subtitle.generic = When password expires you will be locked out of your account. This password is set on another website. Click the button below to go there and set a new password.
+password.expiring.soon.subtitle.specific = When password expires you will be locked out of your {0} account. This password is set on another website. Click the button below to go there and set a new password.
```

Comment:
Are these translations coming from `master`?

Reply:
yup they are coming from this change pr from master: https://github.com/okta/okta-signin-widget/pull/3171/files